be1-go/channel/election/mod_test.go

```diff
@@ -0,0 +1,279 @@
+package election
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/require"
+	"go.dedis.ch/kyber/v3"
+	"go.dedis.ch/kyber/v3/sign/schnorr"
+	"golang.org/x/sync/semaphore"
+	"golang.org/x/xerrors"
+	"io"
+	"os"
+	"path/filepath"
+	"popstellar/channel"
+	"popstellar/crypto"
+	"popstellar/message/messagedata"
+	"popstellar/message/query/method/message"
+	"popstellar/network/socket"
+	"popstellar/validation"
+	"sync"
+	"testing"
+)
+
+const relativeExamplePath string = "../../../protocol/examples/messageData"
+
+func Test_CastVoteAndResult(t *testing.T) {
```

Comment:
As I told the others it is super helpful to have a small comment about tests to explain what they are testing and doing.